src/components/toasts/TestnetToast.js

```diff
@@ -3,16 +3,18 @@ import { Icon } from '../icons';
 import { Nbsp, Text } from '../text';
 import Toast from './Toast';
 import { useInternetStatus } from '@/hooks';
-import { getNetworkObject } from '@/networks';
-import { ChainId } from '@/networks/types';
+import { ChainId } from '@/chains/types';
 import { useConnectedToHardhatStore } from '@/state/connectedToHardhat';
+import { chainsName, chainsNativeAsset } from '@/chains';
 
 const TestnetToast = ({ chainId }) => {
   const { connectedToHardhat } = useConnectedToHardhatStore();
   const isConnected = useInternetStatus();
-  const { name, colors: networkColors } = getNetworkObject({ chainId });
-  const [visible, setVisible] = useState(chainId !== ChainId.mainnet);
-  const [networkName, setNetworkName] = useState(name);
+  const nativeAsset = chainsNativeAsset[chainId];
+  const name = chainsName[chainId];
+  const color = isDarkMode ? nativeAsset.colors.primary : nativeAsset.colors.fallback || nativeAsset.colors.primary;
+  const [visible, setVisible] = useState(true);
```

Comment:
I had left a comment on this same piece of code in a previous PR for the default state to still be `chainId !== ChainId.mainnet` instead of `true` here